src/Main.java

```diff
@@ -1,5 +1,62 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        var dog = 8.0;
+        var cat = 3.6;
+        var paper = 763789;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+
+        var result1 = dog + 4;
+        var result2 = cat + 4;
+        var result3 = paper + 4;
```

Comment:
Тут нужно использовать существующую переменную.
Вот так
```suggestion
        dog = dog + 4;
        cat = cat + 4;
        paper = paper + 4;
```
В остальных задачках аналогично.